analytics-listeners/src/main/java/org/exoplatform/analytics/listener/social/AnalyticsActivityTagsListener.java

```diff
@@ -81,6 +83,24 @@ private void addEventStatistic(Set<TagName> tagNames, String activityId) {
     }
   }
 
+  private void addEventDocumentsStatistic(Set<TagName> tagNames, TagObject tagObject) {
```

Comment:
No documents in Meeds-io, please consider making this extensible to add specific behavior in other addons if needed.